smtp/smtpc.h

```diff
@@ -116,6 +116,7 @@ struct smtpc_request {
 	int status;
 	/** Error message. */
 	const char *reason;
+	char *smtp_error;
```

Comment:
Perhaps you can name as 'error' because structure used for smtp request already contains the prefix 'smtp' and we can omit prefix 'smtp' for variable with error. I will not insist.

Reply:
Fixed.